### DISCUSSION TITLE
Filament assignment drop down missing Manufacturer

### DISCUSSION THREAD

**Discussion by mnebelung** [OctoFarm/OctoFarm#750] [Feature Request]
The filament assignment drop downs in the UI are missing manufacturer (unlike the Filament Manager plugin in OctoPrint), as a result it's hard to know which spool to assign if you have two of the same color and type.  E.g. below I have two white spools of PETG so the only way to tell them apart in the drop down is by weight remaining:
![Screen Shot 2021-09-10 at 10 24 44 AM](https://user-images.githubusercontent.com/16807643/132869861-9c2bc42d-6a0c-4919-b64e-135e8492eff9.png)

Over in OctoPrint, I can see the manufacturer on the drop down label, making it obvious which one I need without having to remember how much weight is remaining:
![image](https://user-images.githubusercontent.com/16807643/132870099-64402441-60af-48fd-8c87-31e47e65d0a4.png)

Perhaps in a perfect world there would be a setting to choose which filament manager fields are shown in the drop down labels by user, giving them choice, but at the least, having this mirror the Filament Manager plugin would a big help.

**Comment by NotExpectedYet** [comment]
This should be an easy fix. I'll make it an issue and have a look into adding it in the next release.

**Comment by NotExpectedYet** [comment]
![image](https://user-images.githubusercontent.com/61137988/137624149-56ab86fb-c5be-4a85-ab23-74fab13c6e2e.png)

Will be in V1.2

**Reply by NotExpectedYet** [reply]
#761 for tracking
